src/assets/styles/pages/Homepage.styles.js

```diff
@@ -0,0 +1,226 @@
+  position: relative;
+  background-image: url("${({ imageSource }) => imageSource}");
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  &::after {
+    position: absolute;
+    width: 70px;
+    height: 70px;
+    content: '';
+    clip-path: polygon(0 0, 16% 0, 16% 84%, 100% 84%, 100% 100%, 0 100%);
+    background-color: ${({ theme }) => theme.color.beige};
+    left: -20px;
+    bottom: -20px;
+  }
+`;
+
+export const AdvantagesSection = styled(StyledSection)``;
+export const ServicesSection = styled(StyledSection)``;
+
+export const HighlightedHeading = styled.h2`
+  font-size: ${({ theme }) => theme.font.size.headingSmall};
+  position: relative;
+
+  &::before {
+    z-index: -1;
+    position: absolute;
+    content: '';
+    width: 100%;
+    height: calc(${({ theme }) => theme.font.size.headingSmall} * 1.5);
+    top: calc(-${({ theme }) => theme.font.size.headingSmall} / 6);
+    left: -50%;
+    background-color: ${({ theme }) => theme.color.beige};
+  }
+`;
+
+export const StyledList = styled.ul`
+  padding: 0;
+  list-style: none;
+
+  li {
+    margin: 30px 0;
+
+    h3 {
+      font-family: ${({ theme }) => theme.font.family.montserrat};
+      font-weight: 700;
+      margin: 0;
+    }
+  }
+`;
+
+export const StyledButton = styled.button`
+  border: 1px solid ${({ theme }) => theme.color.black};
+  background-color: transparent;
+  font-size: ${({ theme }) => theme.font.size.button};
+  font-family: ${({ theme }) => theme.font.family.montserrat};
+  padding: 10px 15px;
+  font-weight: 500;
+  text-transform: uppercase;
+  display: block;
+  margin: ${({ isCentered }) => (isCentered ? '15px auto' : '15px 0')};
+`;
+
+export const StyledLinkButton = styled(Link)`
+  font-family: ${({ theme }) => theme.font.family.montserrat};
+  font-size: ${({ theme }) => theme.font.size.paragraph};
+  color: ${({ theme }) => theme.color.black};
+  text-decoration: underline;
+  position: relative;
+
+  &::after {
+    position: absolute;
+    content: '';
+    background-image: url("${arrowIcon}");
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: 0 50%;
+    width: 20px;
+    height: 20px;
+    right: -35px;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+`;
+
+export const ShowcaseSection = styled(StyledSection)`
+  h2 {
+    font-size: ${({ theme }) => theme.font.size.headingMobile};
+    text-align: center;
+    margin: 30px 0;
+  }
+`;
+
+export const ShowcaseGallery = styled.div`
+  margin: 50px 0;
+`;
+
+export const ShowcaseImage = styled.img`
+  width: 100%;
+  height: ${({ isBig }) => (isBig ? '250px' : '125px')};
+  object-fit: cover;
+  margin: 10px 0;
+`;
+
+export const TeamSection = styled(StyledSection)``;
+
+export const TeamImage = styled.div`
+  margin-top: 30px;
+  width: 90%;
+  height: 300px;
+  position: relative;
+  background-image: url("${({ imageSource }) => imageSource}");
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  &::after {
```

Comment:
Those corners created in ::after and ::before pseudoelements can be extracted into styled components function stored in theme.js